dev/docker/ranger/ranger-dependency.sh

```diff
@@ -24,7 +24,7 @@ ranger_dir="$(cd "${ranger_dir}">/dev/null; pwd)"
 # Environment variables definition
 RANGER_VERSION=2.4.0
 RANGER_PACKAGE_NAME="ranger-${RANGER_VERSION}-admin.tar.gz" # Must export this variable for Dockerfile
-RANGER_DOWNLOAD_URL=https://github.com/datastrato/apache-ranger/releases/download/release-ranger-${RANGER_VERSION}/ranger-${RANGER_VERSION}-admin.tar.gz
+RANGER_DOWNLOAD_URL=https://github.com/apache/apache-ranger/releases/download/release-ranger-${RANGER_VERSION}/ranger-${RANGER_VERSION}-admin.tar.gz
```

Comment:
@xunliu can you please confirm that it is a correct change?

Reply:
That change is probably incorrect, but we may need to change how we obtain apache ranger. Why are we not getting it from teh ASF repo?

Reply:
> That change is probably incorrect, but we may need to change how we obtain apache ranger. Why are we not getting it from teh ASF repo?

Apache Ranger didn't provide a release download, https://github.com/apache/ranger/releases.
So we compiled it myself and uploaded it to the datastrato/apache-ranger repository.

Reply:
GitHub is not an official release area for ASF projects. The official releases are put on https://dist.apache.org/. For Ranger see https://dist.apache.org/repos/dist/release/ranger/.

Reply:
Reverting back for now until we can find a better way of doing this